spec/features/create_new_collection_spec.rb

```diff
@@ -43,14 +43,22 @@
       expect(page).to have_content('Depositing')
 
       # Simulate the deposit process
-      CollectionVersion.last.tap do |version|
-        version.collection.update!(druid: 'druid:dc224fz4940')
-        version.deposit_complete!
-      end
+      collection_version = CollectionVersion.last
+      collection_version.collection.update!(druid: 'druid:dc224fz4940')
+      collection_version.deposit_complete!
 
       # This link appears when depositing is complete
       click_link "Edit #{name}"
       expect(page).not_to have_content('Depositing')
+
+      # Cancel editing to confirm we get back to the collection settings view page
```

Comment:
Can we revert this test? I think that using a feature spec to test link resolution will unnecessarily slow the test suite.  This effectively tests that the browser follows links, which I believe is something we can take for granted.  If we want to add a test, how about checking that the link exists in the component test, which is much faster.

Reply:
Just removed the feature test.  No current component tests for the affected components ... none added at this point.